src/app/components/confirmation/confirmation.component.css

```
@@ -0,0 +1,97 @@
+/* Modal Content */
+.modal-content {
+  color: #ffff;
+  border: 0;
+  background: #371B1B;
+  border-radius: 15px 15px 0 15px;
```

Нижний левый угол у модального окна скруглен. Это видно, когда удерживаешь кнопку нет. В других модальных окнах это скругление на мой взгляд уместно, потому что там нет кнопки снизу слева, а здесь от этого скругления думаю стоит избавиться. Можно вот так исправить
```suggestion
  border-radius: 15px 15px 0 0x;
```